test/functional/page_objects/dashboard_page.js

```diff
@@ -299,8 +299,9 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
 
       await PageObjects.header.waitUntilLoadingHasFinished();
 
-      // Confirm that the Dashboard has been saved.
-      return await testSubjects.exists('saveDashboardSuccess');
+      // Confirm that the Dashboard has been saved and close the toast.
+      await testSubjects.existOrFail('saveDashboardSuccess');
+      await find.clickByCssSelector('[data-test-subj="saveDashboardSuccess"] [data-test-subj="toastCloseButton"]');
```

Comment:
Does using `await testSubjects.click('saveDashboardSuccess toastCloseButton');` work?  I see that style used somewhere else.

Reply:
Good call, according to the tests for the test subj package, that should work.